### PR TITLE
Send and Receive data from the Api

### PR DESCRIPTION
**Pull request by Yerimah**
In this branch we:

- Created a new game with the name of your choice by using the API.
- Implemented the "Refresh" button.
- Implement the form "Submit" button.
- Use async and await JavaScript features to consume the API

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -20,23 +20,23 @@ <h2>Recent Scores</h2>
...
                 <ul class="scorelist">
-                    <li class="score">James: 100</li>
+                    <!-- <li class="score">James: 100</li>
                     <li class="score">Tim: 200</li>
                     <li class="score">David: 120</li>
                     <li class="score">Henry: 211</li>
                     <li class="score">John: 210</li>
                     <li class="score">Thomas: 100</li>
-                    <li class="score">Gerald: 100</li>
+                    <li class="score">Gerald: 100</li> -->
```
Comment (Dipeshtwis) [inline src/index.html]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+const refreshbtn = document.querySelector('.refresh-btn');
+
+refreshbtn.addEventListener('click', () => {
+  const items = document.querySelector('.scorelist');
+
+  while (items.firstChild) {
+    items.removeChild(items.firstChild);
+  }
+});
```
Comment (Dipeshtwis) [inline src/index.js]: - [ ] When the Refresh button is click, it should hit the GET API endpoint causing the scores list to change but in your case currently, it removing all content from the UI. Please fix that.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Yerimah, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used `async`, `await`, and `fetch` method to send and receive data.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] README file should have a step to run the project locally. Currently, you have missed mentioning the command to run the project. [Hint: `npm start`]

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @Yerimah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**